src/core/components/hv-navigator/index.tsx

```diff
@@ -154,6 +154,7 @@ export default class HvNavigator extends PureComponent<Props> {
             cardStyleInterpolator: isModal
               ? NavigatorService.CardStyleInterpolators.forVerticalIOS
               : undefined,
+            gestureEnabled: false,
```

Comment:
Shouldn't we condition based on `isModal`? I assume this also disables the swipe right, which we wouldn't want. Also only iOS should be allowed to swipe right on stacks